HIP/hip-850.md

```diff
@@ -0,0 +1,69 @@
+---
+hip: 850
+title: Enhancing Supply Key Functionality for NFT Updates in Treasury Wallet
```

Comment:
Can we use the word "account" instead of "wallet"? Wallet is ambiguous, it sometimes is used to mean account, and sometimes is used to represent the wallet software (i.e. HashPack, Blade, MetaMask, etc)

Reply:
Change made